docker-compose.yaml

```
@@ -119,6 +132,8 @@ services:
     volumes:
       - ./services/kubernetesmisc/src:/app/services/kubernetesmisc/src
       - ./node-packages:/app/node-packages:delegated
+    ports:
+      - '9008:9229'
```

```suggestion
      - '9007:9229'
```
Should be 9007 as per https://github.com/amazeeio/lagoon/pull/1925/files#diff-3090e25fed17a6bd57433c6bbadc451aR95